pom.xml

```diff
@@ -458,6 +458,18 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
```

Comment:
Missing version of the plugin. You have to specify the version (ideally in the properties) and use it here.